lensterxyz/lenster
# feat: image cropper

## What does this PR do?

Fixes #76
Close ENG-194

Add cropping feature when uploading a profile picture

## Type of change


## How should this be tested?

Go to Profile settings > Upload avatar, select a file, crop it and press Upload and save, verify the new profile picture
